handle_latest_data.go

```diff
@@ -68,18 +68,49 @@ func handleLatestData(
 
 	debugf("* hosts identifiers: %s", identifiers)
 
-	params := Params{
-		"hostids": identifiers,
-	}
+	var (
+		items     []Item
+		webchecks []HTTPTest
+	)
 
-	var items []Item
 	err = withSpinner(
-		":: Requesting information about hosts items",
+		":: Requesting information about hosts items & web scenarios",
 		func() error {
-			items, err = zabbix.GetItems(params)
-			return err
+			errs := make(chan error, 0)
+
+			go func() {
+				var err error
+
+				items, err = zabbix.GetItems(Params{
+					"hostids":  identifiers,
+					"webitems": "1",
+				})
+
+				errs <- err
+			}()
+
+			go func() {
+				var err error
+
+				webchecks, err = zabbix.GetHTTPTests(Params{
+					"hostids":     identifiers,
+					"expandName":  "1",
+					"selectSteps": "extend",
+				})
+
+				errs <- err
+			}()
+
+			for _, err := range []error{<-errs, <-errs} {
```

Comment:
I am talking about the second part of `range`, why there is such ambigious statement as `[]error{<-errs, <-errs}`, why not just `range errs`?